packages/collector/test/tracing/database/redis/test.js

```diff
@@ -22,18 +22,32 @@ const {
 const ProcessControls = require('../../../test_util/ProcessControls');
 const globalAgent = require('../../../globalAgent');
 
-// v3 is treated as the legacy version. It does not support Redis clustering or the @redis/client package.
-// Clustering support was introduced in v4:
-// https://github.com/redis/node-redis/blob/master/CHANGELOG.md#v400---24-nov-2021
+// v3 is considered the legacy version.
+// - It does not support Redis clustering.
+// - It does not support the newer `@redis/client` package.
+// Clustering support was officially introduced in v4
+// Redis Sentinel support was added in v5.
 const legacyVersion = 'v3';
 
-// Please run this command on the root folder to start the redis instance:
-// node bin/start-test-containers.js --redis
-//
-// Please set the environment variables to run the tests against azure redis cluster:
-//    export AZURE_REDIS_CLUSTER=team-nodejs-redis-cluster-tekton.redis.cache.windows.net:6380
-//    export AZURE_REDIS_CLUSTER_PWD=
-['default', 'cluster'].forEach(setupType => {
+/**
+ * Supported Redis setups for local testing:
```

Comment:
qs: I was wondering if we should add "runSingleType" because I assume you do not start all types at once? 

```js
let setupTypes = ['default', 'cluster', 'sentinel']
const singleSetupType = false; 
setupTypes = [singleSetupType] || setupTypes

setupTypes.forEach